src/librustc/mir/interpret/mod.rs

```diff
@@ -40,27 +39,6 @@ use ty::codec::TyDecoder;
 use std::sync::atomic::{AtomicU32, Ordering};
 use std::num::NonZeroU32;
 
-#[derive(Clone, Debug, PartialEq, Eq, Hash, RustcEncodable, RustcDecodable)]
```

Comment:
cc @oli-obk, @RalfJung

Reply:
Yeah this is dead, seems I forgot to remove it.